### PR TITLE
[Tachyon-22] make worker web page compatible with java 8

### PR DESCRIPTION
**Pull request by siyangy**
Any direct or indirect reference to Map$Entry will cause the jsp page stop working with java8:
PWC6033: Error in Javac compilation for JSP
PWC6199: Generated servlet error:
The type java.util.Map$Entry cannot be resolved. It is indirectly referenced from required .class files

This PR is mostly a temp workaround that eliminates all the reference to it.
The long term solution should be updating to Jetty 9.2 which will officially support java 8: https://webtide.com/jetty-9-2-0-released/.

### CODE REVIEW

**Comment by yupeng9** [comment]
LGTM. Thanks for adding this workaround

**Comment by yupeng9** [comment]
@siyangy is this still needed after the exclusion of the dependencies from https://github.com/Alluxio/alluxio/pull/2696 ?

**Comment by siyangy** [comment]
This is actually to fix a separate problem: Java 8 cannot work with jsp with reference to Map$Entry. While the exclusion is to make test work.

File: core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java [hunk 1]
```diff
@@ -27,13 +27,17 @@
 import alluxio.worker.block.BlockWorker;
 import alluxio.worker.block.meta.BlockMeta;
 
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
```
Comment (apc999) [inline core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java]: I think now org and com imports are in the same session with new Alluxio code style.

File: core/server/src/main/java/alluxio/web/WebInterfaceWorkerGeneralServlet.java [hunk 2]
```diff
@@ -19,11 +19,14 @@
 import alluxio.worker.block.BlockStoreMeta;
 import alluxio.worker.block.BlockWorker;
 
+import com.google.common.collect.Lists;
+
```
Comment (apc999) [inline core/server/src/main/java/alluxio/web/WebInterfaceWorkerGeneralServlet.java]: same here.

File: core/server/src/main/java/alluxio/web/WebInterfaceWorkerGeneralServlet.java [hunk 3]
```diff
@@ -133,6 +136,51 @@ public String getWorkerAddress() {
 
   }
 
+  /**
+   * A wrapper class of the usage info per tier for displaying in the UI.
```
Comment (apc999) [inline core/server/src/main/java/alluxio/web/WebInterfaceWorkerGeneralServlet.java]: Could you point to the JIRA number here in comment?

**Comment by apc999** [comment]
@siyangy thanks for the fix! I left some minor comments.

File: core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java (hunk 1 above)
Comment (siyangy) [inline core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java]: are you suggesting removing the blank line? There isn't any checkStyle error.

Comment (aaudiber) [inline core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java]: unfortunately checkstyle doesn't support this sort of enforcement. In Eclipse you can set "Organize Imports" like [this](http://www.alluxio.org/documentation/resources/eclipse_imports.png) and in Intellij you can set the import code style like [this](http://www.alluxio.org/documentation/resources/intellij_imports.png).

File: core/server/src/main/java/alluxio/web/WebInterfaceWorkerGeneralServlet.java (hunk 2 above)
Comment (siyangy) [inline core/server/src/main/java/alluxio/web/WebInterfaceWorkerGeneralServlet.java]: there's nothing can be combined.

File: core/server/src/main/java/alluxio/web/WebInterfaceWorkerGeneralServlet.java (hunk 3 above)
Comment (siyangy) [inline core/server/src/main/java/alluxio/web/WebInterfaceWorkerGeneralServlet.java]: done

File: core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java (hunk 1 above)
Comment (siyangy) [inline core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java]: thanks Andrew, done.

**Comment by yupeng9** [comment]
LGTM

File: core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java (hunk 1 above)
Comment (apc999) [inline core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java]: fix import here?

Comment (siyangy) [inline core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java]: Isn't it already fixed in my new commit?

**Comment by giaosudau** [comment]
Please release it soon.

**Pull request merged**